client/core/messaging.js

```diff
@@ -33,11 +33,12 @@ function messenger () {
   function onmessage (messageHandler) {
```

Comment:
Github was just collapsing the lines funny--made it look like onmessage was nested in onready.